### PR TITLE
feat: port scripts to ts

### LINKED ISSUE #46: Port repo scripts to TypeScript

### PR DESCRIPTION
**Pull request by jedrichards**
Refs #57 
Resolve #46

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,15 +9,12 @@ jobs:
       - node/install:
           install-npm: false
           install-yarn: false
-          node-version: '12.10.0'
+          node-version: '13.8.0'
```
Comment (christianklotz) [inline .circleci/config.yml]: I always wonder if there is a way to get the value from `.nvmrc`? I guess the answer is no?

Comment (jedrichards) [reply .circleci/config.yml]: Yeah agreed, that would be nice. I think we just have to keep an eye on Circle's Node Orb, and perhaps it'll get smart enough to do this. I notice they do mention nvm in its source code, but don't seem to be using it in the way you suggest. Perhaps we could make a contribution?

https://circleci.com/orbs/registry/orb/circleci/node

**Comment by jedrichards** [comment]
Ah you mean because the schema are in some senses the "source"? Yeah I see what you mean, although in another way they are, like you say, static assets for the TypeScript source - which is what is compiled to become the actual published package code 🤷‍♂

**Comment by christianklotz** [comment]
Yeah, exactly. So I'm not 💯 sure what should be considered source here – though my gut feeling tells me it's the schema. If there's no more elegant alternative to the Current setup I'm happy to run with this one though.

**Comment by jedrichards** [comment]
I'm comfortable how it is at the moment, since it's a bit six of one, half a dozen of the other. We can see how it sits for a while. It doesn't affect the public api of the package which remains just `import fileFormat from '@sketch-hq/sketch-file-format`.

File: .circleci/config.yml (hunk 1 above)
Comment (christianklotz) [reply .circleci/config.yml]: Sounds good, can't find the repo though 🤔 Would be good to file an issue before contributing

**Pull request merged**